engine/Scripts/local_build_win.ps1

```diff
@@ -0,0 +1,8 @@
+param(
```

Comment:
how to use this file?

Reply:
Run command in Readme.md
`.\local_build_win.ps1`